src/get-env-variables.js

```diff
@@ -0,0 +1,60 @@
+const exec = require('child_process').exec
+// import spawn from 'spawn-command-with-kill'
```

Comment:
Ah, totally forgot to mention this in my comment! Spawn wasn't working... I pulled it out into a separate file and ran it and everything works just fine. Do you know why that might be?